db/database.go

```diff
@@ -119,6 +119,15 @@ type DatabaseContext struct {
 	ServeInsecureAttachmentTypes bool                     // Attachment content type will bypass the content-disposition handling, default false
 	NoX509HTTPClient             *http.Client             // A HTTP Client from gocb to use the management endpoints
 	ServerContextHasStarted      chan struct{}            // Closed via PostStartup once the server has fully started
+	Scopes                       map[string]Scope         // A map keyed by scope name containing a set of scopes/collections.
+}
+
+type Scope struct {
+	Collections map[string]Collection
+}
+
+type Collection struct {
+	Datastore base.Bucket // Where data is stored for the collection - This should be changed to allow multiple collections to share the same underlying SDK connection once support for >1 collection is added.
```

Comment:
Is this being used anywhere, or is databaseContext.Bucket still being used for SDK connections?